templates/questions/show.html.twig

```diff
@@ -34,6 +34,24 @@
                                     </div>
                                 </form>
                             </div>
+                            {# Remove tags #}
+                            {% if is_granted('EDIT', question) %}
+                                <form action="{{ path('app_question_publish', { slug: question.slug }) }}" method="POST">
+                                    <div class="mt-2">
+                                        <h3>Tags:</h3>
```

Comment:
Move all of these tag links outside of the form.

Reply:
Done